### PR TITLE
add test for a new vxlan default rule/meter

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy [hunk 1]
```diff
@@ -114,7 +114,7 @@ class MetersSpec extends HealthCheckSpecification {
         expect: "Only the default meters should be present on each switch"
         switches.each { sw ->
             def meters = northbound.getAllMeters(sw.dpId)
-            assert meters.meterEntries.size() == 2
+            assert meters.meterEntries.size() == 3
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy]: As far as I understand this should remain '2' for non-noviflow switch

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy [hunk 2]
```diff
@@ -173,6 +173,10 @@ class SwitchRulesSpec extends HealthCheckSpecification {
                         [
                                 installRulesAction: InstallRulesAction.INSTALL_ROUND_TRIP_LATENCY,
                                 cookie            : Cookie.ROUND_TRIP_LATENCY_RULE_COOKIE
+                        ],
+                        [
+                                installRulesAction: InstallRulesAction.INSTALL_UNICAST_VXLAN,
+                                cookie            : Cookie.VERIFICATION_UNICAST_VXLAN_RULE_COOKIE
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy]: Again, this is not expected to work on _all_ switches.

Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchRulesSpec.groovy]: thx, fixed
needed `assumeFalse` was added

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy (hunk 1 above)
Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy]: thx, fixed

**Pull request merged**